contributors/devel/pull-requests.md

```diff
@@ -1,329 +1 @@
-
-We might still ask you to clean up your commits at the very end for the sake of a more readable history, but don't do this until asked: typically at the point where the PR would otherwise be tagged `LGTM`.
-
-Each commit should have a good title line (<70 characters) and include an additional description paragraph describing in more detail the change intended.
-
-**General squashing guidelines:**
-
-* Sausage => squash
-
- Do squash when there are several commits to fix bugs in the original commit(s), address reviewer feedback, etc. Really we only want to see the end state and commit message for the whole PR.
-
-* Layers => don't squash
-
- Don't squash when there are independent changes layered to achieve a single goal. For instance, writing a code munger could be one commit, applying it could be another, and adding a precommit check could be a third. One could argue they should be separate PRs, but there's really no way to test/review the munger without seeing it applied, and there needs to be a precommit check to ensure the munged output doesn't immediately get out of date.
-
-A commit, as much as possible, should be a single logical change.
-
-## 7. KISS, YAGNI, MVP, etc.
-
-Sometimes we need to remind each other of core tenets of software design - Keep It Simple, You Aren't Gonna Need It, Minimum Viable Product, and so on. Adding a feature "because we might need it later" is antithetical to software that ships. Add the things you need NOW and (ideally) leave room for things you might need later - but don't implement them now.
-
-## 8. It's OK to Push Back
-
-Sometimes reviewers make mistakes. It's OK to push back on changes your reviewer requested. If you have a good reason for doing something a certain way, you are absolutely allowed to debate the merits of a requested change. Both the reviewer and reviewee should strive to discuss these issues in a polite and respectful manner.
-
-You might be overruled, but you might also prevail. We're pretty reasonable people. Mostly.
-
-Another phenomenon of open-source projects (where anyone can comment on any issue) is the dog-pile - your PR gets so many comments from so many people it becomes hard to follow. In this situation, you can ask the primary reviewer (assignee) whether they want you to fork a new PR to clear out all the comments. You don't HAVE to fix every issue raised by every person who feels like commenting, but you should answer reasonable comments with an explanation.
-
-## 9. Common Sense and Courtesy
-
-No document can take the place of common sense and good taste. Use your best judgment, while you put
-a bit of thought into how your work can be made easier to review. If you do these things your PRs will get merged with less friction.
+This file has been moved to [/contributors/guide/pull-requests.md](/contributors/guide/pull-requests.md).
```

Comment:
Tombstone files should have an end date.

e.g.:
```
<!--
This file is a placeholder to preserve links.  Please remove after 3 months or the release of kubernetes 1.10, whichever comes first.
-->
```

Reply:
Noted. It's mainly that all the links in other k/subrepos are in need of updating in the meantime.